index.html

```diff
@@ -0,0 +1,179 @@
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- main programe section -->
+      <section class="main-programe">
+        <h3 class="sec-title">Main Programe</h3>
+        <article class="cards-group">
+          <div class="card">
+            <img src="./images/home/research.svg" alt="icon" />
+            <h4 class="card-title">Lecture</h4>
+            <p class="card-desc">
+              listen to the speakers from various countries about the message of
+              sharing and opening.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/design.svg" alt="icon" />
+            <h4 class="card-title">CC Exhibition</h4>
+            <p class="card-desc">
+              Appreciate various creations applying CC licence of
+              artists,organized from Art center Nabi.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/networking.svg" alt="network-icon" />
+            <h4 class="card-title">Forum</h4>
+            <p class="card-desc">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/discussion.svg" alt="icon" />
+            <h4 class="card-title">Workshop</h4>
+            <p class="card-desc">
+              Try creating your own work using open source licence rather than
+              just watching it.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/knowledge.svg" alt="icon" />
+            <h4 class="card-title">CC Ignite</h4>
+            <p class="card-desc">
+              Get opportuntities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </article>
+        <a class="more" href="#">See the whole programe</a>
+      </section>
+      <section class="feature-section">
+        <h3 class="sec-title">Feature speakers</h3>
+        <ul class="speak-group"></ul>
+      </section>
+
+      <!-- partner section -->
+      <section class="partners">
+        <h3 class="sec-title">Partner</h3>
+        <ul class="partner-list">
+          <li><img src="./images/partners/airbnb.svg" alt="airbnb" /></li>
+          <li><img src="./images/partners/google.svg" alt="google" /></li>
+          <li>
+            <img src="./images/partners/att-vector-logo.svg" alt="partners" />
+          </li>
+          <li><img src="./images/partners/mozilla.svg" alt="mozilla" /></li>
+          <li><img src="./images/microsoft-logo.svg" alt="logo" /></li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <img src="./images/microsoft-logo.svg" alt="logo" />
+        <p>© Microsoft 2022</p>
+      </div>
+    </footer>
+    <script src="./apps/app.js"></script>
+  </body>
```

Comment:
@Shedrack-Sunday 
![Screenshot 2023-04-18 170517](https://user-images.githubusercontent.com/115831124/232772053-82eb25b7-f5a9-40d0-8952-8c476b84be0e.png)
as you can see micro verse said you can personalize it on your own so, in my opinion, no need for this to change its IP on your own and other demands you required I have done all buttons and all other requirements. Otherwise, if new reviewer I know the need to always require changes and make things tough so all other requirements are done. But thankyou for such consideration.